decapode/kafka_integration.py

```diff
@@ -0,0 +1,75 @@
+from typing import Awaitable, Optional
+import json
+import logging
+import requests
+
+from aiohttp import web
+from kafka import KafkaConsumer, KafkaProducer
+
+from decapode.config import KAFKA_URI, KAFKA_API_VERSION, DECAPODE_API_URL, DATAGOUV_API_URL
+from decapode.crawl import crawl_urls
+
+log = logging.getLogger("decapode-kafka")
+
+def produce(producer: KafkaProducer, id: str, data: dict) -> None:
+    key = id.encode("utf-8")
+
+    value = {
```

Comment:
We'll need to discuss the message + topic based on our signal target architecture